keyboards/planck/keymaps/vaire/keymap.c

```diff
@@ -0,0 +1,254 @@
+
+ #include "planck.h"
+ #include "action_layer.h"
+
+
+extern keymap_config_t keymap_config;
+
+#define EXCEL_HOME LALT(KC_H)
+#define EXCEL_COL LCTL(KC_SPC)
+#define EXCEL_ROW LSFT(KC_SPC)
+#define DESK_LEFT LCTL(LGUI(KC_LEFT))
+#define WIN_LEFT LSFT(KC_TAB)
+#define WIN_CLOSE LALT(KC_F4)
+#define WIN_RIGHT KC_TAB
+#define DESK_RIGHT LCTL(LGUI(KC_RGHT))
+#define DOCS LGUI(KC_E)
+#define SNAP_RIGHT LGUI(KC_RGHT)
+#define SNAP_UP LGUI(KC_UP)
+#define SNAP_LEFT LGUI(KC_LEFT)
+#define SNAP_DOWN LGUI(KC_DOWN)
+#define COPY LCTL(KC_C)
+#define PASTE LCTL(KC_V)
+#define CUT LCTL(KC_X)
+#define UNDO LCTL(KC_Z)
+#define FIND LCTL(KC_F)
+
+
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+
+
+  KEYMAP(
```

Comment:
Please replace all of the `KEYMAP`s with `LAYOUT_ortho_4x12` or `LAYOUT_planck_grid`.